src/components/Profile/ProfileEdit/ProfileEdit.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../../assets/css/pixels';
+@import '../../../assets/css/colors';
+
+.ProfileEdit {
+  display: inline-block;
+  .modal-content {
+    padding-left: 15px !important;
+    box-shadow: none;
+    padding-left: 0;
+    padding-right: 0;
+    background-color: initial;
+  }
+}
+
+@media screen and (max-width: 760px) {
+  .ProfileEdit {
+    .modal-content {
+      width: 90%;
```

Comment:
set max-width, this may be very big on large screens, remember .container has max-width of 1100px, is modal-content is on more than 1100px maay look unexpectedly

Reply:
good, thanks.